<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TopToolWidget</class>
 <widget class="QWidget" name="TopToolWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>342</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>342</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>342</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TopToolWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="FilletWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>380</width>
       <height>342</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>380</width>
       <height>342</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:4px;
background-color: #FCFCFC;</string>
     </property>
     <widget class="QWidget" name="TopWidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>380</width>
        <height>50</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>380</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>380</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#TopWidget{border-bottom:1px solid #DDDDDD;border-bottom-left-radius:0px;border-bottom-right-radius:0px;}</string>
      </property>
      <widget class="QLabel" name="labelTilte">
       <property name="geometry">
        <rect>
         <x>19</x>
         <y>14</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 16px &quot;微软雅黑&quot;;
color:#333333;</string>
       </property>
       <property name="text">
        <string>在线客服</string>
       </property>
      </widget>
      <widget class="QPushButton" name="closeButton">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>12</y>
         <width>32</width>
         <height>28</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>28</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{border-image: url(:/ZcloudDesk/image/close.png);border-radius:none;background-color: rgba(255, 255, 255, 0);}

QPushButton:hover,pressed{border-image: url(:/ZcloudDesk/image/close_sel.png);};</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>380</width>
        <height>250</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>380</width>
        <height>250</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>380</width>
        <height>250</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QScrollBar::vertical{width:8px;background:rgba(0,0,0,0%);margin:0px,0px,0px,0px;}QScrollBar::handle:vertical{width:8px;background:rgba(0,0,0,25%); border-radius:4px;min-height:20;}
QScrollBar::handle:vertical:hover{width:8px;background:rgba(0,0,0,50%); border-radius:4px;min-height:20;}
QScrollBar::add-line:vertical{height:9px;width:8px;border-image:url(:/images/a/3.png);subcontrol-position:bottom;}QScrollBar::sub-line:vertical{height:9px;width:8px;border-image:url(:/images/a/1.png);subcontrol-position:top;}QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical{background:rgba(0,0,0,10%);border-radius:4px;};</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>380</width>
         <height>250</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <property name="horizontalSpacing">
           <number>20</number>
          </property>
          <property name="verticalSpacing">
           <number>12</number>
          </property>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>92</x>
        <y>305</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/ZcloudDesk/image/icon_service.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>118</x>
        <y>305</y>
        <width>112</width>
        <height>15</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>112</width>
        <height>15</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>112</width>
        <height>15</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14px &quot;微软雅黑&quot;;
color:#666666;</string>
      </property>
      <property name="text">
       <string>航天信息客服热线</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>238</x>
        <y>305</y>
        <width>49</width>
        <height>15</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>49</width>
        <height>15</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>49</width>
        <height>15</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 14px &quot;微软雅黑&quot;;
color:#1F8BED;</string>
      </property>
      <property name="text">
       <string>95113</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ZcloudDesk.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>TopToolWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>-5</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
